internal/configs/configschema/coerce_value.go

```diff
@@ -57,7 +58,15 @@ func (b *Block) coerceValue(in cty.Value, path cty.Path) (cty.Value, error) {
 
 	attrs := make(map[string]cty.Value)
 
-	for name, attrS := range b.Attributes {
+	// Stable sort keys for consistent error reports
```

Comment:
The second instance of this makes sense, but in this case no errors are possible, so I'd leave it out.

Reply:
I don't follow, attrs are the main error path in the linked ticket?

Reply:
my bad, github collapsed the relevant part

Reply:
Spoke with @cube2222, simple misunderstanding.